test-integration/mocha-fork/package.json

```
@@ -0,0 +1,20 @@
+{
+  "name": "mocha-test",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "npx mocha --exit --timeout 10000"
```

`npx` isn't necessary here because `npm run test` takes care of adjusting the `PATH`.

```suggestion
    "test": "mocha --exit --timeout 10000"
```